tooling/mocks.yml

```diff
@@ -8,18 +8,19 @@ networks:
 
 services:
   repository:
-    image: httpd:2.4
+    image: nginx:1.16.0
```

Comment:
Can we use the latest `1.17` version (let's skip patch version)?

Reply:
Nope, this one is last stable